### ISSUE TITLE
"Draft" message appears at each post

### ISSUE THREAD

**Issue by dimobelov** [bludit/bludit#842]
https://github.com/bludit/bludit/blob/3510fa6e30416e79e0dfd85f76f6553c606c99c7/bl-kernel/admin/views/edit-content.php#L75

Fix:
```
	<?php if($page->draft()): ?>
	<div class="alert alert-warning p-1 mt-1 mb-0"><?php $L->p('the-content-is-saved-as-a-draft-to-publish-it') ?></div>
	<?php endif; ?>
```

**Comment by dignajar** [comment]
Fixed, thank you!

**Comment by dignajar** [comment]
Fixed, please check the new version 3.4.0.
